tests/Transfer/resources/nhost/aaa.sql

```diff
@@ -0,0 +1,44 @@
+--
+-- PostgreSQL database cluster dump
```

Comment:
Lets rename this file also using the pattern `1_file_name`, `2_next_file` and so on.

Lets also make sure all the sql files follow this convention so we know in which order they are supposed to run / execute and so the files have a meaningful name